.github/workflows/ruby.yml

```diff
@@ -0,0 +1,50 @@
+name: Ruby Gem
+
+on: [push]
```

Comment:
Is this what we want to do? Maybe this is appropriate for this repo, but at least for others, I'd expect to use on PR actions to trigger tests and then version on a PR closed where the PR is merged.

As I said, may well be appropriate here.

Reply:
Well, we don't have any other actions…

Reply:
`on:
  pull_request:
    types: [assigned, opened, labeled, unlabeled, synchronize, closed]
`
Is what we're likely going to use in the smartly-ui repo.